rest/runProxyMethod.ts

```diff
@@ -0,0 +1,50 @@
+import { RestManager } from "../bot.ts";
+import { SnakeCasedPropertiesDeep } from "../types/util.ts";
+import { RestRequestRejection, RestRequestResponse } from "./rest.ts";
+
+export type ProxyMethodResponse<T> = Omit<RestRequestResponse | RestRequestRejection, "body"> & { body?: T };
+
+// Left out proxy request, because it's not needed here
+// this file could also be moved to a plugin.
+export async function runProxyMethod<T = any>(
```

Comment:
Where and how would this be used?

Reply:
This function would be used in your proxy, just like here: https://github.com/discordeno/discordeno/blob/main/template/bigbot/src/rest/mod.ts#L48